automation/active_user_input/english_working-groups-config.yml

```
@@ -80,10 +85,11 @@ wg-diversity-inclusion:
 #    - channels.md
     - chat-platform-inclusivity.md
     - documentation-accessibility.md
-#    - documentation_discoverability.md
+    - documentation_discoverability.md
```

```suggestion
    - documentation-discoverability.md
```
Recently Renamed:
https://github.com/chaoss/wg-dei/blob/main/focus-areas/project-and-community/documentation-discoverability.md

---

Thanks for fixing this!